docs/root/start/sandboxes/win32_front_proxy.rst

```diff
@@ -0,0 +1,352 @@
+Windows based Front proxy
+=========================
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+To get a flavor of what Envoy has to offer on Windows, we are releasing a
```

Comment:
i think the same as on the install page that this needs to warn the user that this is not open source software and that the image is very large

Reply:
Let's reach a consensus in #16380 and I will mirror it.

Reply:
resolving as discussed on #16380